packages/nuxt-locokit/package.json

```diff
@@ -25,27 +26,32 @@
     "@locokit/designsystem": "^0.8.0",
     "@locokit/i18n": "0.8.0",
     "@locokit/sdk": "^0.8.0",
-    "@nuxt/kit": "^3.0.0-rc.13",
-    "@pinia/nuxt": "^0.4.2",
-    "@vee-validate/rules": "^4.7.2",
-    "pinia": "^2.0.23",
-    "primevue": "^3.18.1",
-    "tailwindcss": "^3.2.2",
-    "vee-validate": "^4.7.2",
+    "@feathersjs/authentication-client": "^5.0.0-pre.32",
+    "@feathersjs/feathers": "^5.0.0-pre.32",
+    "@feathersjs/rest-client": "^5.0.0-pre.32",
+    "@nuxt/kit": "^3.1.1",
+    "@pinia/nuxt": "^0.4.6",
+    "@tailwindcss/line-clamp": "^0.4.2",
+    "@vee-validate/rules": "^4.7.3",
+    "bootstrap-icons": "^1.10.3",
+    "pinia": "^2.0.30",
+    "primevue": "^3.23.0",
+    "tailwindcss": "^3.2.4",
+    "vee-validate": "^4.7.3",
     "vue-i18n": "^9.3.0-beta.6"
   },
   "devDependencies": {
     "@nuxt/module-builder": "latest",
-    "@nuxtjs/eslint-config-typescript": "^11.0.0",
-    "@nuxtjs/tailwindcss": "^6.1.3",
-    "@typescript-eslint/eslint-plugin": "^5.40.0",
-    "@typescript-eslint/parser": "^5.40.0",
+    "@nuxtjs/eslint-config-typescript": "^12.0.0",
+    "@nuxtjs/tailwindcss": "^6.3.1",
+    "@typescript-eslint/eslint-plugin": "^5.50.0",
+    "@typescript-eslint/parser": "^5.50.0",
     "@vue/eslint-config-typescript": "^11.0.2",
     "eslint": "^8.25.0",
-    "eslint-plugin-nuxt": "^3.2.0",
-    "eslint-plugin-vue": "^9.6.0",
-    "nuxt": "^3.0.0-rc.13",
-    "prettier": "^2.7.1",
-    "sass": "^1.54.9"
+    "eslint-plugin-nuxt": "^4.0.0",
+    "eslint-plugin-vue": "^9.9.0",
+    "nuxt": "^3.1.1",
+    "prettier": "^2.8.3",
```

Comment:
Same thoughts here. I think it could be better to centralize those dependencies at the root level.
I see an advantage : all packages shared the same dependencies + rules for linting. (and for typescript too)

Reply:
Cf https://github.com/locokit/locokit/pull/187#discussion_r1116859903